### ISSUE TITLE
Problem in PDF generation if we use url and callout

### ISSUE THREAD

**Issue by epabion** [quarto-dev/quarto-cli#184]
When whe try to generate a pdf from quatro's default we get error on generating because the Blue color is not definined. This is due to the hypersetup's option in which the `citecolor` and the `urlcolor` are `Blue`, because LaTeX can be case sensitive on some systems `blue` must be used.

**Comment by dragonstyle** [comment]
It seems that perhaps that is intentional (the default colors are coming from the pandoc default template for LaTeX). It appears that it is expecting to use valid color options from xcolor (https://ctan.org/pkg/xcolor). `Blue` is a valid `dvipsname` so I would expect it to work. Is there something else potentially causing the issue here?

In addition, if you'd like, you can use `urlcolor` in the front matter to set the color to one of your liking.

**Comment by epabion** [comment]
When I use a minimal document ([LinkTest.qmd.txt](https://github.com/quarto-dev/quarto-cli/files/7534489/LinkTest.qmd.txt)) it works,  but when I try to add callout ([MNWE.qmd.txt](https://github.com/quarto-dev/quarto-cli/files/7534522/MNWE.qmd.txt)) there is a problem and I get the following message error : `compilation failed- error Package xcolor Error: Undefined color 'Blue'.`

**Comment by dragonstyle** [comment]
This is a great clue - there is definitely an issue related to the way we're loading xcolors (and not giving it options) when using tcolorbox to render callouts. Would you mind sharing the tex output of the example that is failing for you? Thank you!

**Comment by epabion** [comment]
You will find the tex output [MNWE.tex.txt](https://github.com/quarto-dev/quarto-cli/files/7534765/MNWE.tex.txt) of [MNWE.qmd.txt](https://github.com/quarto-dev/quarto-cli/files/7534767/MNWE.qmd.txt) in attachement. I add the `.txt` extension so that the files can be attached by github.

**Comment by epabion** [comment]
We can see in [MNWE.tex.txt](https://github.com/quarto-dev/quarto-cli/files/7534867/MNWE.tex.txt) which didn't compile fine that the package `color` is loaded in line 22. If we load the package `xcolor` with the same options, all gets fine.

**Comment by dragonstyle** [comment]
Ok, so the mystery is that xcolor does need those options passed to it when loaded (and tcolorbox and its dependencies ultimately do load xcolor). The mystery here is why those options aren't being set. When I render, the top of my tex looks like

```
% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
  letterpaper,
  DIV=11]{scrartcl}
```

which is what I'd expect from looking at the pandoc default latex template.  This is very different from the latex that you shared, and explains why it is working for me and not for you. Is there anything special about the way you're rendering or the pandoc version (e.g. using a non default template or special version of pandoc)? You could check the version of pandoc that quarto sees using `quarto pandoc --version` - I'm using 2.16.1.

**Comment by dragonstyle** [comment]
Hi @epabion just a quick nudge to see if you have any additional information that might help me track this down. If not, completely understand...

**Comment by epabion** [comment]
Sorry for the late reply.
>
> Ok, so the mystery is that xcolor does need those options passed to it when loaded (and tcolorbox and its dependencies ultimately do load xcolor). The mystery here is why those options aren't being set. When I render, the top of my tex looks like
> % Options for packages loaded elsewhere
> \PassOptionsToPackage{unicode}{hyperref}
> \PassOptionsToPackage{hyphens}{url}
> \PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
> %
> \documentclass[
>   letterpaper,
>   DIV=11]{scrartcl}
>
> which is what I'd expect from looking at the pandoc default latex template. This is very different from the latex that you shared,  and explains why it is working for me and not for you. Is there anything special about the way you're rendering or the pandoc version (e.g. using a non default template or special version of pandoc)? You could check the version of pandoc that quarto sees using quarto pandoc --version - I'm using 2.16.1.

I thing that I have find the reason. I have some templates in ~/.pandoc/templates. In this repository I have a file nammed default.latex in which there is the following line : \PassOptionsToPackage{usenames,dvipsnames}{color} % color is loaded by hyperref. If I remove the file default.latex and do quarto render MNVE.qmd --to pdf all gets fine. I have to update my old default.latex file in ~/.pandoc/templates. For the completness I'm using quarto version 0.2.275 in which I have apply the fix e080a00 that you mentionned in the issue #183 by modified the file /share/quarto/share/filters/quarto-pre/quarto-pre.lua and version 2.16.1 of pandoc.
